doc/source/working-with-s3.rst

```diff
@@ -1,11 +1,21 @@
-Tutorial: Working with S3
-=========================
+Working with S3
+===============
 
 This is a simple guide that demonstrates how to use TileDB on S3-compatible
 backends. We will first explain how to set up an AWS account and configure
 TileDB for `AWS S3 <https://aws.amazon.com/s3/>`_, and then show how to
 set up and use `minio <https://minio.io>`_ locally.
 
+.. note::
+
+  After setting up TileDB to work with AWS S3 or minio, all your program
```

Comment:
Not sure this should be a separate `note` block, also "all your program" -> "your TileDB programs"